packages/components/src/components/hds/app-side-nav/index.hbs

```diff
@@ -27,7 +27,7 @@
       />
     {{/if}}
 
-    <div class="hds-app-side-nav__wrapper-body">
+    <div class="hds-app-side-nav__wrapper-body" {{did-insert this.didInsertNavWrapperBody}}>
```

Comment:
[suggestion] my understanding is that we're moving away from `ember-render-modifiers` in favour of custom modifiers (see Dylan's PR https://github.com/hashicorp/design-system/pull/2856/files for example); maybe something we can do here as well?

Reply:
or is it because you didn't want to have to do also the migration of the `did-insert` applied to the root element of this component? in which case makes sense, and I have no objection to keep it "old school" for now :)

Reply:
Hahah I can update to use the new modifier pattern! Might as well since I'm touching the file.